app/assets/stylesheets/types.scss

```diff
@@ -0,0 +1,34 @@
+// Place all the styles related to the Types controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+@mixin type_button($back_coulour, $colour) {
+  background-color: $back_coulour;
+  color: $colour;
+  border-radius: 25px;
+  border: 5px solid $back_coulour;
+  font: 100% Lucida Console;
+  text-decoration: none;
+  display:block;
+}
+
+.type-button-normal { @include type_button(#DAF7A6, black); }
```

Comment:
You could create these with a @mixin loop if you want to 😉
mixins can use a list of variables, so that you can define the name+color for each class